src/lib/mod.rs

```diff
@@ -0,0 +1,93 @@
+#![deny(unsafe_code)]
+#![allow(
+    clippy::must_use_candidate,
+    clippy::missing_panics_doc,
+    clippy::missing_errors_doc,
+    clippy::module_name_repetitions
+)]
+pub mod color;
+pub mod matcher;
+use lazy_static::lazy_static;
+use std::{borrow::Borrow, path::Path};
+
+lazy_static! {
```

Comment:
These would also be detestably more performant as _just_ `const` or `static` vars. 

`const` variables will be inlined, `static` will be by reference. `lazy_static` adds a check every time to see if thy are initialized. 

`for` loops aren't `const` able yet though, so if clarity of code is the key feature, I think you've nailed that :+1: